map_server/CMakeLists.txt

```diff
@@ -55,6 +55,7 @@ target_link_libraries(map_server
 )
 
 add_executable(map_server-map_saver src/map_saver.cpp)
+add_dependencies(map_server-map_saver ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
```

Comment:
This is catkin specific. `catkin_EXPORTED_TARGETS` will not exist in an ament build